addons/addon-base-raas/packages/base-raas-services/lib/data-source/__tests__/environment-resource-service.test.js

```diff
@@ -494,6 +494,23 @@ describe('EnvironmentResourceService', () => {
 
   describe('addToKmsKeyPolicy', () => {
     it('add new principal to KMS policy with no principals', async () => {
+      environmentResourceService._settings = {
```

Comment:
Why are these needed now but weren't needed earlier?

Reply:
Because there is an environmentResourceService setup in the beforeEach but it didn't include getBoolean so I needed to have the implementation of the mocked getBoolean function. Putting it here overrode the beforeEach one and meant that the environmentResourceService mock this test was using didn't have the methods that the beforeEach one did. Thanks for pointing this out and making me figure this out though--I can just put the getBoolean mock in the beforeEach one which reduces code repetition and still works!